planet_explorer/tests/Data/test_add_to_map/planet_orders/5c9e6c59-eb35-485d-ab7d-04a75e9e0f14/Add_to_map_test_QGIS/files/PSScene/20221221_084022_18_2414/analytic_sr_udm2/20221221_084022_18_2414_3B_AnalyticMS_metadata.xml

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Instead of having a long path `planet_explorer/tests/Data/test_add_to_map/planet_orders/5c9e6c59-eb35-485d-ab7d-04a75e9e0f14/Add_to_map_test_QGIS/files/PSScene/20221221_084022_18_2414/20221221_084022_18_2414_metadata.json`
we should a simple directory access like
`planet_explorer/tests/data/planet_orders/20221221_084022_18_2414_metadata.json` to all other test data files.